src/modules/user/service.js

```diff
@@ -37,8 +42,31 @@ const getUserById = (id) => {
   return db.User.findByPk(id);
 };
 
+const validatePhoneNumber = (value, helpers) => {
+  const condition = value.match(
+    `^\\+(${ALLOWED_PHONE_NUMBER_COUNTRY_CODE.join('|')})[0-9]{6,10}$`,
+  );
+  return condition ? value : helpers.message(PHONE_NUMBER_IS_INVALID_MESSAGE);
+};
+
+const updateUser = async (user, newData) => {
+  try {
+    await db.User.update({ ...newData }, { where: { id: user.id } });
+    const newUser = await getUserById(user.id);
+    newUser.password = undefined;
```

Comment:
You can use `exclude` option in sequelize for excluding password or other attributes from the response